src/ensembl/package.json

```diff
@@ -27,9 +27,9 @@
     "storybook": "start-storybook -p 9001 -c .storybook",
     "deploy-docs": "./deploy-docs.sh",
     "check-types": "tsc",
-    "jest": "jest",
-    "test": "jest --verbose",
-    "test:watch": "jest --watch",
+    "jest": "jest --maxWorkers=1",
```

Comment:
Why do we have this npm script anyway? Doesn't the current implementation of `npm test` make it redundant?

Reply:
I used it to get the console output as Jest didn't print them in verbose mode. Now that Jest doesn't have that issue anymore, it is redundant indeed.